### PR TITLE
Rollout improved example converter

### PR DESCRIPTION
**Pull request by t0yv0**
Rolls out improved TF example converter.

### CODE REVIEW

File: provider/cmd/pulumi-resource-hcloud/schema.json [hunk 1]
```diff
@@ -3367,7 +3367,7 @@
...
...rverPublicNetArgs.builder()\n                .ipv4Enabled(true)\n                .ipv6Enabled(true)\n                .build())\n            .build());\n\n    }\n}\n```\n```yaml\nresources:\n  # Create a new server from the snapshot\n  fromSnapshot:\n    type: hcloud:Server\n    name: from_snapshot\n    properties:\n      name: from-snapshot\n      image: ${packerSnapshot.id}\n      serverType: cx11\n      publicNets:\n        - ipv4Enabled: true\n          ipv6Enabled: true\nvariables:\n  # Get image infos because we need the ID\n  packerSnapshot:\n    fn::invoke:\n      Function: hcloud:getImage\n      Arguments:\n        withSelector: app=foobar\n        mostRecent: true\n```\n\u003c!--End PulumiCodeChooser --\u003e\n\n## Primary IPs\n\nWhen creating a server without linking at least one ´primary_ip´, it automatically creates \u0026 assigns two (ipv4 \u0026 ipv6).\nWith the public_net block, you can enable or link primary ips. If you don't define this block, two primary ips (ipv4, ipv6) will be created and assigned to the server automatically.\n\n## Import\n\nServers can be imported using the server `id`:\n\n```sh\n$ pulumi import hcloud:index/server:Server myserver id\n```\n\n",
```
Comment (flostadler) [inline provider/cmd/pulumi-resource-hcloud/schema.json:3370]: Seems like some of the examples got removed here? Why's that?

File: provider/cmd/pulumi-resource-hcloud/schema.json [hunk 2]
```diff
@@ -3920,7 +3920,7 @@
...
+            "description": "Provides a Hetzner Cloud SSH key resource to manage SSH keys for server access.\n\n## Import\n\nSSH keys can be imported using the SSH key `id`:\n\n```sh\n$ pulumi import hcloud:index/sshKey:SshKey mykey id\n```\n\n",
```
Comment (flostadler) [inline provider/cmd/pulumi-resource-hcloud/schema.json:3923]: Here we lost all examples except the import?

**Review by flostadler — COMMENTED** [review]
I might miss something, but it seems like sometimes examples got dropped

File: provider/cmd/pulumi-resource-hcloud/schema.json (hunk 2 above)
Comment (t0yv0) [reply provider/cmd/pulumi-resource-hcloud/schema.json]: This is back now.

File: provider/cmd/pulumi-resource-hcloud/schema.json (hunk 1 above)
Comment (t0yv0) [reply provider/cmd/pulumi-resource-hcloud/schema.json:3370]: This is back now.

**Review by flostadler — APPROVED** [review]

**Pull request merged**